### PR TITLE
CTCTRADERS-1129: New UI to show NoReleaseForTransit  status

### PR DESCRIPTION
**Pull request by ashwinirudrappa**

### CODE REVIEW

File: migrations/applied_migrations/NoReleaseForTransit.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+echo "Adding routes to conf/app.routes"
+echo "" >> ../conf/app.routes
+echo "GET        /noReleaseForTransit                       controllers.NoReleaseForTransitController.onPageLoad" >> ../conf/app.routes
+
+echo "Adding messages to conf.messages"
+echo "" >> ../conf/messages.en
+echo "noReleaseForTransit.title = noReleaseForTransit" >> ../conf/messages.en
+echo "noReleaseForTransit.heading = noReleaseForTransit" >> ../conf/messages.en
+
+echo "Migration NoReleaseForTransit completed"
```
Comment (ericohwotu) [inline migrations/applied_migrations/NoReleaseForTransit.sh:15]: is this file needed?

**Review by ericohwotu — APPROVED** [review]

File: migrations/applied_migrations/NoReleaseForTransit.sh (hunk 1 above)
Comment (ashwinirudrappa) [reply migrations/applied_migrations/NoReleaseForTransit.sh:15]: No, we don't need it, will remove Thanks
